CONTRIBUTING.md

```diff
@@ -0,0 +1,177 @@
+# Contributing
+
+- [Contributing](#contributing)
+  - [Types of Contributions](#types-of-contributions)
+    - [Report Bugs](#report-bugs)
+    - [Fix Bugs](#fix-bugs)
+    - [Implement Features](#implement-features)
+    - [Submit Feedback](#submit-feedback)
+  - [Get Started!](#get-started)
+  - [Pull Request Guidelines](#pull-request-guidelines)
+  - [Tips](#tips)
+    - [Generating code from Markdown](#generating-code-from-markdown)
+    - [Generate code from Markdown and vice versa](generate-code-from-markdown-and-vice-versa)
+    - [Generate code from Markdown on commit](#generate-code-from-markdown-on-commit)
+  - [New release](#new-release)
+
+Contributions are welcome, and they are greatly appreciated! Every little bit
```

Comment:
We can create an empty contributing.md and point there. We will update the contributing guide during the sprint.